src/stores/configurationStore.ts

```diff
@@ -125,11 +125,11 @@ export const useConfigurationStore = defineStore('configuration', {
     },
 
     /** Fetches the configuration from the web server and, if successful, triggers some actions. */
-    loadConfiguration (): Promise<any> {
+    loadConfiguration (env = import.meta.env): Promise<any> {
       // need to return a promise because action is called via dispatch
       return new Promise((resolve) => {
-        this.setConfiguration(import.meta.env)
-        this.setSessionVariables(import.meta.env)
```

Comment:
Unfortunately, when testing locally, this included ALL my terminal's environment variables. (Enter `env` to see what I mean.) So, for testing, now I pass in just the env object I want.